certbot/compat/filesystem.py

```diff
@@ -112,6 +160,7 @@ def open(file_path, flags, mode=0o777):  # pylint: disable=redefined-builtin
         # We set third parameter to 0 (`False`) to say that this security descriptor is
         # NOT constructed from a default mechanism, but is explicitly set by the user.
         # See https://docs.microsoft.com/en-us/windows/desktop/api/securitybaseapi/nf-securitybaseapi-setsecuritydescriptordacl  # pylint: disable=line-too-long
+        security.SetSecurityDescriptorOwner(user, False)
```

Comment:
1. nit: this line should go before the comment, because it explains what the next line does
2. do I understand correctly that this is something we probably wanted to add previously that was caught by fixing up the checks here, and not a new change?

Reply:
1. Done
2. Yes, you are right, I forgot to add this before.